tests/uploader_test.py

```diff
@@ -328,6 +328,14 @@ def test_upload_large(self):
         self.assertEqual(resource3["tags"], ["upload_large_tag", UNIQUE_TAG])
         self.assertEqual(resource3["resource_type"], "raw")
 
+        # should allow fallback of upload large with remote url to regular upload
+        resource4 = uploader.upload_large("http://cloudinary.com/images/old_logo.png", tags=[UNIQUE_TAG])
```

Comment:
make this url const, something like REMOTE_TEST_IMAGE, i would even mock this test, but it's OK.